src/App.js

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
-
+import { Tabs } from './components/Tabs/Tabs';
 import './App.scss';
 
 /* eslint-disable */
 const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  { id: 'tab-1', title: 'Home', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Profile', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Contact', content: 'Some text 3' },
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    tabsCopy: tabs.map(tab => ({ ...tab })),
```

Comment:
значения в стейте должны быть простыми
```suggestion
    tabs: tabs.map(tab => ({ ...tab })),
```